packages/amplify_authenticator/example/integration_test/sign_up_with_username_test.dart

```diff
@@ -45,106 +48,69 @@ void main() {
 
     // Scenario: Login mechanism set to "username"
     testWidgets('Login mechanism set to "username"', (tester) async {
-      await tester.loadAuthenticator(authenticator);
-      await tester.navigateToSignUp();
-
-      // Then I see "Username" as an input field
-      expect(
-        // Match on the title of the username field since it is dependent on
-        // the username alias selected.
-        find.descendant(
-          of: find.byKey(keyUsernameSignUpFormField),
-          matching: find.text('Username'),
-        ),
-        findsOneWidget,
-      );
+      SignUpPage signUpPage = SignUpPage(tester: tester);
+      SignInPage signInPage = SignInPage(tester: tester);
+      await loadAuthenticator(tester: tester, authenticator: authenticator);
+      await signInPage.navigateToSignUp();
+      signUpPage.expectUserNameIsPresent();
     });
 
     // Scenario: "Preferred Username" is included from `aws_cognito_signup_attributes`
     testWidgets(
-        '"Preferred Username" is included from `aws_cognito_signup_attributes`',
```

Comment:
The back ticks were causing the following error when running the test individually from VS code.

```
No tests match regular expression "^sign-in-with-username "Preferred Username" is included from ( \(variant: .*\))?$".
```